### ISSUE TITLE
Moderation enhancement: big pause button for all chat

### ISSUE THREAD

**Issue by brittag** [lazerwalker/azure-mud#323]
As a virtual community space where we could have large jumps in chat traffic or high-volume chaotic responses in the chat space, it could be helpful to have a big pause button for chat, as a chance to take a breath and reset the tone.

Example implementation: the mod on duty could push a "trip the breakers" button that would make it say "live chat is temporarily paused, please enjoy the talks in the meantime" in all rooms.

**Comment by kawa-kitsuragi** [comment]
Also worth considering maybe - Discord style slow mode, which forces any individual user to wait a specified amount of time between messages?

**Comment by lazerwalker** [comment]
Yep, that's consistent both with what was discussed in the 7/18 meeting, and I think would be more natural to attendees (it's also what Twitch offers). Updating the title accordingly!

We already have logic to rate-limit `/shout` commands per-player, so adapting that to work with regular messages would be fairly straight-forward for someone newer to the codebase who wanted to take it on. The bulk of the work will be exposing an admin toggle to enable/disabled it.
